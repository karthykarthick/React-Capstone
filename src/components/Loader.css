.loader-container {
  display: flex;
  justify-content: center;
}

.loader-container img {
  max-width: 700px;
}
