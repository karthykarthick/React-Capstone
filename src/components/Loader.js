import React from 'react';
import loader from '../assets/images/loader.gif';
import './Loader.css';

const Loader = () => (
  <div className="loader-container">
    <img
      alt="Loader gif"
      src={loader}
    />
  </div>
);

export default Loader;
