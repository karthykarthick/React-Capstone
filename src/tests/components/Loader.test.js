import ShallowRenderer from 'react-test-renderer/shallow';
import React from 'react';
import Loader from '../../components/Loader';

const renderer = new ShallowRenderer();

it('should render Loader correctly', () => {
  renderer.render(<Loader />);

  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
