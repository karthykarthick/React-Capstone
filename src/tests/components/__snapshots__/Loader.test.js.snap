// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Loader correctly 1`] = `
<div
  className="loader-container"
>
  <img
    alt="Loader gif"
    src="loader.gif"
  />
</div>
`;
